worldedit-bukkit/src/main/java/com/boydti/fawe/bukkit/wrapper/AsyncChunk.java

```diff
@@ -169,7 +170,7 @@ public boolean unload() {
 
     @Override
     public boolean isSlimeChunk() {
-        return false;
+        return world.getChunkAt(x, z).isSlimeChunk();
```

Comment:
This wasn't really needed since AsyncChunk will only be accessed by FAWE but I'm with with it still being here.